src/main/java/com/cburch/logisim/fpga/hdlgenerator/InlinedHdlGeneratorFactory.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Logisim-evolution - digital logic design tool and simulator
+ * Copyright by the Logisim-evolution developers
+ *
+ * https://github.com/logisim-evolution/
+ *
+ * This is free software released under GNU GPLv3 license
+ */
+
+package com.cburch.logisim.fpga.hdlgenerator;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+import com.cburch.logisim.data.AttributeSet;
+import com.cburch.logisim.fpga.data.MappableResourcesContainer;
+import com.cburch.logisim.fpga.designrulecheck.Netlist;
+import com.cburch.logisim.fpga.designrulecheck.NetlistComponent;
+
+public class InlinedHdlGeneratorFactory implements HDLGeneratorFactory {
+
+  @Override
+  public boolean GenerateAllHDLDescriptions(Set<String> HandledComponents, String WorkingDir,
```

Comment:
Method name spelling, argument names spelling (multiple cases)

Reply:
This I will not yet change in this PR, as HDLGeneratorFactory is "everywhere". Refactoring of HDLGeneratorFactory will be a seperate PR